### DISCUSSION TITLE
Plans for Google Paystore ?

### DISCUSSION THREAD

**Discussion by erd82** [razzo04/rhasspy-mobile-app#11] [General]
Is this app planned for Google Playstore? Background: I am using a managed android smartphone and am not allowed to install apks there.

**Comment by razzo04** [comment]
Maybe at some point, I may publish on Google Playstore but not in the near future.

**Comment by erd82** [comment]
Hi @razzo04, thanks for your feedback!

**Comment by Doni49** [comment]
There is a Rhasspy App for android on the Google Play store.  It says the author's name is Killian E.
